packages/cloud_firestore/ios/Classes/CloudFirestorePlugin.m

```diff
@@ -46,8 +46,6 @@ - (FlutterError *)flutterError {
       query = [query queryWhereField:fieldName isGreaterThan:value];
     } else if ([op isEqualToString:@">="]) {
       query = [query queryWhereField:fieldName isGreaterThanOrEqualTo:value];
-    } else if ([op isEqualToString:@"array-contains"]) {
-      query = [query queryWhereField:fieldName arrayContains:value];
```

Comment:
Curious why this is going away? Is there going to be a replacement?

Reply:
Not sure, but I couldn't find the symbol for the `arrayContains` in latest